scvi/external/solo/_model.py

```diff
@@ -185,13 +194,32 @@ def train(
             plan_kwargs.update(update_dict)
         else:
             plan_kwargs = update_dict
+
```

Comment:
check if callbacks is in **kwargs, if it isn't put an empty list there and then add the ES callback there. if it is, add ES to the existing list